integration-test/ERC721Mintable/erc721mintable.readContract.test-integration.ts

```diff
@@ -127,4 +127,30 @@ describe('E2E Test: Sdk (read)', () => {
       expect(tokenMetadata).toStrictEqual(expectedTokenMetadata);
     });
   });
+
+  describe('As an account I should get list of collections that i have created', () => {
```

Comment:
aren't we missing an integ test for the cases where the address is not valid or missing?

Reply:
it's done on the unit tests